CHANGELOG.md

```diff
@@ -1,40 +1,73 @@
 # Mattermost Desktop Application Changelog
 
-## UNDER DEVELOPMENT
+## Release v3.6.0
 
-The "UNDER DEVELOPMENT" section of the Mattermost Desktop changelog appears
-in the product's `master` branch to note key changes committed to `master`
-and are on their way to the next stable release.
-When a stable release is pushed, "UNDER DEVELOPMENT" heading is removed
-from the final changelog of the release.
+Release date: February 28, 2017
 
-Release date: TBD
+Upgrading to Mattermost server 3.6 or later is recommended, as new features for the desktop app have been added following the release of the team sidebar.
 
 ### Improvements
-- Added a new team button next to the team tabs
-
-#### All Platforms
-- Suppress white screen which is displayed for a moment on startup
-- Support unread indications in team sidebar of Mattermost server 3.6
-- Remove confusing `Ctrl(CMD)+S` shortcut for searching within a team
+ - Added support for unread indicators following the release of team sidebar in Mattermost server 3.6
+ - Removed a confusing `CTRL/CMD+S` shortcut for searching within a Mattermost team
+ - Added support for SAML OneLogin and Google authentication for Enterprise users
+ - Switching to a server from the system tray icon, from "Window" menu bar item, or through CTRL/CMD+{n} shortcut now works while viewing the Settings page works
```

Comment:
Looks good. I added some tweaks. If they're not wrong, I feel it's ready to merge.